dot/rpc/modules/childstate_test.go

```diff
@@ -138,9 +138,9 @@ func TestChildStateModule_GetKeys(t *testing.T) {
 		},
 	}
 	for _, tt := range tests {
-		res := make([]string, 0)
-		tt.args.res = &res
 		t.Run(tt.name, func(t *testing.T) {
+			res := make([]string, 0)
```

Comment:
nit the same as:

```suggestion
			res := []string{}
```

Also you could move this in your `tests` declaration by using a little helper function like

```go
stringSlicePtr := func(s []string) *[]string { return &s }
```

which you can declare inline in your test before `tests := ...`